<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bilgi_Ekrani</class>
 <widget class="QMainWindow" name="Bilgi_Ekrani">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1413</width>
    <height>744</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>471</width>
      <height>461</height>
     </rect>
    </property>
    <property name="title">
     <string>KİTAP İŞLEMLERİ</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>361</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="kITAPIDLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>KITAP ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="kITAPIDLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="SERINOguncel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>SERI NO</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="sERINOguncel"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="kITAPADILabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>KITAP ADI</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="kITAPADILineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="yAZARADILabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>YAZAR ADI</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="yAZARADILineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="yAYINEVILabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>YAYIN EVI</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="yAYINEVILineEdit"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="kITAPTESLIMLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>KATEGORILER</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="kITAPTESLIMComboBox">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>BİLİM KURGU</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PSİKOLOJİ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EĞİTİM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DÜNYA KLASİKLERİ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TÜRK KLASİKLERİ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FANTEZİ</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="tESLIMTARIHILabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TESLIM TARIHI</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="btn_sil">
        <property name="text">
         <string>SİL</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="btn_guncelle">
        <property name="text">
         <string>GÜNCELLE</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>100</y>
      <width>741</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>490</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>KİTAP ARA</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="SERINOBILGI">
        <property name="text">
         <string>SERI NO</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="sERINObilgi"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>ARA</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>BİLGİ EKRANI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_geri">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GERİ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1413</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
